tests/ignite/engine/test_custom_events.py

```diff
@@ -207,6 +213,12 @@ def foo(engine, _):
     assert ret.filter is not None
     assert event.name in f"{ret}"
 
+    ret = event(once=[1, 10])
```

Comment:
Can you also add a test like here : https://github.com/pytorch/ignite/blob/2639d1c7250d2b4912adc6ab9c2fe09726042eaf/tests/ignite/engine/test_custom_events.py#L405-L432
for `once = [1, 10]`